packages/jellyfish-transaction-builder/src/txn/txn_builder_loans.ts

```diff
@@ -1,36 +1,36 @@
 import {
```

Comment:
It is hard for reviewers to see the file changes from github because I did some code refactoring. I would suggest you read the whole file instead.